spec/e2e/default_install_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper_e2e'
+
+describe 'Default install' do
+  def browser
+    @browser ||= new_browser
+  end
+
+  before(:all) do
+    upload_portal_config('empty.yml')
```

Comment:
Looks like tests are failing because that fixture doesn't exist.

Reply:
Yea in my hubris I pulled this out of draft too soon. I quit virtualbox and started using WSL for Windows and it still isn't quite setup yet so I'm relying on the CI to run these.